SUPPORT.md

```diff
@@ -20,7 +20,7 @@ With the following configuration keys:
 
 - `pkg-url` specifies the package download URL for a given target/version, templated
 - `bin-dir` specifies the binary path within the package, templated (with an `.exe` suffix on windows)
-- `pkg-fmt` overrides the package format for download/extraction (defaults to: `tgz`)
+- `pkg-fmt` overrides the package format for download/extraction (defaults to: `tgz`), check [here](https://docs.rs/binstalk-types/latest/binstalk_types/cargo_toml_binstall/enum.PkgFmt.html) for all supported formats.
```

Comment:
For accessibility reasons, `here` is not a recommended label for a link

I suggest

```suggestion
- `pkg-fmt` overrides the package format for download/extraction (defaults to: `tgz`), check [the documentation](https://docs.rs/binstalk-types/latest/binstalk_types/cargo_toml_binstall/enum.PkgFmt.html) for all supported formats.
```

or

```suggestion
- `pkg-fmt` overrides the package format for download/extraction (defaults to: `tgz`), [check the documentation for all supported formats](https://docs.rs/binstalk-types/latest/binstalk_types/cargo_toml_binstall/enum.PkgFmt.html).
```

I don't know what is the best

Reply:
Thanks, the first one looks good to me.